contrib/cirrus/lib.sh

```diff
@@ -227,6 +236,9 @@ use_netavark() {
     export NETWORK_BACKEND=netavark  # needed for install_test_configs()
     msg "Removing any/all CNI configuration"
     rm -rvf /etc/cni/net.d/*
+    # N/B: The netavark/aardvark-dns packages are still installed and
```

Comment:
Do you mean **CNI** packages?

Reply:
Oh woops, yes I did mean CNI.  I'll fix it, thanks.